lib/grape/router.rb

```diff
@@ -11,6 +11,14 @@ def initialize(pattern, attributes = {})
       end
     end
 
+    def self.normalize_path(path)
```

Comment:
This method had been implemented as private before. Ruby doesn't actually allow for private class methods, so moving it above the private block and taking care of the violation doesn't change anything